pages/docs/reference/delegated-properties.md

```diff
@@ -444,4 +487,21 @@ class C {
 
 </div>
 
-Note that the `provideDelegate` method affects only the creation of the auxiliary property and doesn't affect the code generated for getter or setter.
+Note that the `provideDelegate` method affects only the creation of the auxiliary property and doesn't affect the code
+generated for getter or setter.
+
+With the `PropertyDelegateProvider` interface from the standard library, you can create delegate providers as anonymous
+objects without creating new classes.
```

Comment:
> as anonymous objects without creating new classes.

This note would be also valuable for implementing property delegates with `ReadOnlyProperty` and `ReadWriteProperty` delegates.